sdc/crypto/jwe_helper.py

```diff
@@ -25,8 +25,6 @@ def decrypt(encrypted_token, key_store=None, purpose=None):
             logger.info("Decrypting JWE kid is {}".format(jwe_kid))
 
             private_jwk = key_store.get_private_key_by_kid(purpose, jwe_kid).as_jwk()
-            if not private_jwk:
-                raise MissingKeyException
```

Comment:
This is technically a change to the interface I think? 

Can't imagine it will affect anyone, but this would be a major version change.

Reply:
Though we're on a 0 major version, so I guess it doesn't matter.

Reply:
That's a fair point.  I'll update the changelog to reflect this.
Functionally it shouldn't break anything as it was impossible to hit this line (as it threw a different exception in the get_private_key_by_kid).

Reply:
People may still be relying on that exception being thrown, so it could technically break something.

Reply:
I investigated a bit just to make sure, the `get_private_key_by_kid` function will raise an InvalidTokenException if the key is missing (or if the purpose / kid are invalid). 

I think the better solution here is to raise a MissingKeyException inside KeyStore rather than an InvalidTokenException. Not directly related to this though.